frontend/src/component/menu/Header/OldHeader.tsx

```diff
@@ -242,7 +223,7 @@ const OldHeader: VFC = () => {
                         <StyledLink to={'/search'}>Search</StyledLink>
                         <StyledLink to='/playground'>Playground</StyledLink>
                         <ConditionallyRender
-                            condition={!killInsightsDashboard && !isOss()}
+                            condition={false}
```

Comment:
This does not look correct
```
!killInsightsDashboard && !isOss()
!false && !isOss()
true && !isOss()
!isOss()
```

Reply:
You are replacing with the wrong value `false`. You PR description says that you are replacing with `true`. This `isOSS` is an unused function.